### PR TITLE
{lib}[foss/2015a] GlobalArrays 5.4b (REVIEW)

### PR DESCRIPTION
**Pull request by tvandera**
GlobalArrays toolkit -- http://hpc.pnl.gov/globalarrays/

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/g/GA/GA-5.4b-foss-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+Starting GA v5.4, ARMCI has been deprecated and replaced with Communication
+runtime for Extreme Scale (ComEx). GA uses ComEx to abstract inter-node
+communication operations. The default ComEx ports use MPI --- which makes GA
+and ComEx portable for high-end systems. Most ComEx implementations use on-node
+shared memory for faster intra-node communication."""
+
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+
+block = 'ConfigureMake';
```
Comment (boegel) [inline easybuild/easyconfigs/g/GA/GA-5.4b-foss-2015a.eb]: what is this supposed to do? it's not used afaik, so please remove?

File: easybuild/easyconfigs/g/GA/GA-5.4b-foss-2015a.eb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+source_urls = ['http://hpc.pnl.gov/globalarrays/download/']
+sources = ["%s-%s.tgz" % (name.lower(), version.replace(".", "-")) ]
```
Comment (boegel) [inline easybuild/easyconfigs/g/GA/GA-5.4b-foss-2015a.eb]: use `%(namelower)s`:

```python
+sources = ["%%(namelower)s-%s.tgz" % version.replace(".", "-")]
```

**Comment by boegel** [comment]
@tvandera: we already have an easyconfig for Global Arrays, where we used `GlobalArrays` as a name (since it's more descriptive than `GA` and has less chance of clashing with something else), maybe stick to that name? (see https://github.com/hpcugent/easybuild-easyconfigs/tree/master/easybuild/easyconfigs/g/GlobalArrays).

Also check to align your easyconfig with that one (cfr. description).

**Comment by tvandera** [comment]
I just tested the GlobaArrays easyconfig, and this seems to work fine. I'm closing this PR.

**Comment by boegel** [comment]
@tvandera: this is still useful, since it's a version update?

**Comment by tvandera** [comment]
Reopened the pull request with an easyconfig for GlobalArrays 5.4b based on the existing GlobalArrays easyconfig.

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/5bc4ae757f9fa34d1581 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/81086736deb7903f79e5 for a full test report.

**Comment by boegel** [comment]
going in, thanks @tvandera!

**Pull request merged**